lang/cs/Org.Apache.REEF.IMRU/API/IMRUJobDefinitionBuilder.cs

```diff
@@ -203,6 +211,17 @@ public IMRUJobDefinitionBuilder SetUpdateTaskMemory(int memory)
         }
 
         /// <summary>
+        /// Sets Per Map Configuration
```

Comment:
I am confused....can you explain.....shouldn't it be part of
IMRUJobDefinition and why do we need Set here? ... can u explain in a bit
of detail

On Fri, Aug 28, 2015 at 10:39 AM, Markus Weimer <notifications@github.com>
wrote:

> In lang/cs/Org.Apache.REEF.IMRU/API/IMRUJobDefinitionBuilder.cs
> <https://github.com/apache/incubator-reef/pull/417#discussion_r38224472>:
>
> > @@ -203,6 +211,17 @@ public IMRUJobDefinitionBuilder SetUpdateTaskMemory(int memory)
> >          }
> >
> >          /// <summary>
> > +        /// Sets Per Map Configuration
>
> This is quite confusing, as this is used to set the configuration used to
> instantiate the IPerMapConfiguration on the Driver, which in turn returns
> the configurations used for the mappers, right?
>
> Also, why not make this part of a ConfigurationModule which has a
> OptionalImpl<IPerMapConfiguration>? That could then be bound to a
> Name<Set<IPerMapConfiguration>> to allow for future expansion.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/417/files#r38224472>.
>